data-serving/scripts/convert_data.py

```diff
@@ -0,0 +1,53 @@
+# Lint as: python3
```

Comment:
I thought that was a google linter thing? I've never seen that outside google

Reply:
oops, removed